lib/supportFeature.js

```diff
@@ -1,22 +1,27 @@
 const features = require('./features')
 
-function versionIndex (versionString) {
-  const versionList = require('minecraft-data').versions.pc
-  for (let i = 0; i < versionList.length; i++) {
-    if (versionList[i].minecraftVersion === versionString) {
-      return versionList.length - i
-    }
-  }
-
-  return -1
-}
+const versionList = require('minecraft-data').versions.pc
+const versionToIndex = Object.fromEntries(versionList.map((version, i) => [version.minecraftVersion, versionList.length - i]))
+const nameToFeature = Object.fromEntries(features.map(feature => [feature.name, feature]))
 
 module.exports = (feature, minecraftVersion) => {
-  const f = features.find(f => f.name === feature)
+  const f = nameToFeature[feature]
```

Comment:
Can't we just have:
```js
module.exports = (featureName, minecraftVersion) => {
  const feature = nameToFeature[featureName]
```
Having a single character variable is not descriptive and commonly seen as bad pattern

Reply:
yes fixed